Yoast/YoastSEO.js
# Add WP-style transliterations

## Summary

This PR can be summarized in the following changelog entry:

* Improves the recognition of keywords with special diacritics in the URL.

## Relevant technical choices:

* A second transliteration map was added in order to match the one that is used by [WP](https://github.com/Yoast/wordpress-develop-mirror/blob/master/src/wp-includes/formatting.php). This transliteration map is much less specific than the one that we have been using so far, but it is used to create automatic permalinks. This way we can match with both language-specific transliterations (that were already available in YoastSEO.js) and with the straightforward  WP way.

## Test instructions

This PR can be tested by following these steps:

* Take a text in a language that contains a lot of accent characters (i.e., any letters from the latin alphabet that additionally have upper elements such as accents, circles, dots, hooks, or lower elements such as tails).
* Make one of the words that contains accent characters your keyword and use it in the title. 
* Check if in the transliteration that appeared in the permalink (automatically, after publication) all accent characters were substituted by their ASCII analogs.
* Check if the URL-Keyword assessment returns "The focus keyword appears in the URL for this page."

Fixes #1607
